models/mart/customer_status_daily_mrr.sql

```diff
@@ -0,0 +1,65 @@
+{{
+    config(
+        materialized='table'
+    )
+}}
+
+
+with customer_daily_mrr as (
+	SELECT 
+		customer_id,
+		customer_name,
+	    mrr_date, 
+	    mrr,
+	    stripe_account,
+	    CASE 
+            WHEN mrr > 0 THEN 1
+            ELSE 0
+            END AS active_day
```

Comment:
Maybe as a one liner?

```suggestion
	    case when mrr > 0 then 1 else 0 end as active_day
```